### PR TITLE
docs: fix links

### LINKED ISSUE #91: docs: links broken
2 links are broken (possibly more):

Where? [here](https://formidable.com/open-source/renature/docs/#documentation)
Which links?

- Core Concepts
- API

Core Concepts is currently https://formidable.com/open-source/renature/docs/'./core-concepts/ and should be https://formidable.com/open-source/renature/docs/core-concepts/

API is currently https://formidable.com/open-source/renature/docs/'./api/ but should be https://formidable.com/open-source/renature/docs/api/

image

### PR DESCRIPTION
**Pull request by ZenaMel**
fixes #91 (not tested)

### CODE REVIEW

File: docs/content/README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ order: 0
 We hope you love `renature` and enjoy learning more about the library! Check out the following sections to get started:
 
 - [**Getting Started**](./getting-started.md) – Learn how to install `renature`, use your first hook, and control animation play states.
-- [**Core Concepts**]('./core-concepts.md) – Interested in the underlying physics and math powering `renature`? Curious to understand the core of how `renature` looks? This section will dive deep into library internals for users and contributors alike.
-- [**API**]('./api.md) – The formal API documentation for `renature`. Complete with live code examples powered by [`react-live`](https://github.com/FormidableLabs/react-live).
+- [**Core Concepts**](core-concepts.md) – Interested in the underlying physics and math powering `renature`? Curious to understand the core of how `renature` looks? This section will dive deep into library internals for users and contributors alike.
```
Comment (parkerziegler) [inline docs/content/README.md:17]: I'd prefer to keep all paths prefixed with `./` to keep the directory structure clear.

```suggestion
- [**Core Concepts**](./core-concepts.md) – Interested in the underlying physics and math powering `renature`? Curious to understand the core of how `renature` looks? This section will dive deep into library internals for users and contributors alike.
```

The main bug seems to be an errant `'` 😅 Same for the link below.

**Comment by parkerziegler** [comment]
Thanks @Zenahr! Just one small suggested fix and then LGTM!

File: docs/content/README.md (hunk 1 above)
Comment (ZenaMel) [reply docs/content/README.md:17]: Ah I see. Makes sense.

**Comment by boygirl** [comment]
@parkerziegler Just a reminder that this docs fix won't get auto-deployed when merged since it's based on a fork. CI won't have access to secrets.

**Comment by parkerziegler** [comment]
Thanks for the reminder @boygirl. I'll have a fast follow PR after this is merged to upgrade a few things on our docs site (`react-static` version, `renature` version, etc.) so that it can get deployed. Did we ever figure out an alternative for community PRs on other projects?

**Comment by boygirl** [comment]
Unfortunately no. Maybe GH actions will have a better solution we can try when that org changes comes into effect

**Comment by ZenaMel** [comment]
I'll unsubscribe for now since my work here seems done ^^

**Pull request merged**